xla/service/gpu/gemm_rewriter.cc

```diff
@@ -698,20 +698,23 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
       return F8ScaleD(instr, existing_gemm, d_scale);
     }
 
-    // Attempt to match approximate GELU activation
-    // (https://arxiv.org/abs/1606.08415), where:
-    // approx_gelu(x) = x * cdf(x)
-    // cdf(x) = 0.5 * (1 + tanh(sqrt(2 / pi) * (x + 0.044715 * x**3))
-    HloInstruction *cdf, *slice_or_bitcast = nullptr;
-    if (Match(instr, m::MultiplyAnyOrder(
-                         m::AnyOf<HloInstruction>(
-                             m::Slice(&slice_or_bitcast,
-                                      CublasLtMatmulMaybeF8(&existing_gemm)),
-                             m::Bitcast(&slice_or_bitcast,
-                                        CublasLtMatmulMaybeF8(&existing_gemm)),
-                             CublasLtMatmulMaybeF8(&existing_gemm)),
-                         m::Op(&cdf).WithOneUser())) &&
-        Match(cdf,
+    {
```

Comment:
do we need to include swish fusion into HandleDot()? @pemeliya

Reply:
ok, it seems this is an element-wise function https://flax.readthedocs.io/en/v0.5.3/_autosummary/flax.linen.swish.html so yes, `HandleMutiply()` is all you need